packages/cli/test/unit/discover/import-discovered-model.test.js

```diff
@@ -48,9 +48,8 @@ describe('importDiscoveredModel', () => {
     expect(modelData.properties)
       .to.have.property('createdAt')
       .deepEqual({
-        type: 'Date',
-        tsType: 'Date',
-        required: false,
```

Comment:
why is this property removed?

Reply:
Good catch!

Model properties are optional by default. Specifying `required: false` is a bit redundant, therefore we are omitting that metadata from the scaffolded code. This behavior is already implemented by `lb4 model` and `lb4 import-lb3-models`, my change is bringing it to `lb4 discover` too, which is good for consistency.